### ISSUE TITLE
Cannot launch this app

### ISSUE THREAD

**Issue by robingranite** [lwouis/alt-tab-macos#372]
Thanks Lwouis! This is a wonderful app I used everyday!

However, today this app(alt-tab-macos version 4.1.2) cannot start in a new MacBook. No icon displayed on status bar. Cannot find this app in Activity Monitor. I tried both downloading app directly and making the app by using homebrew. Same version works fine in an old MacBook.

Both new and old MacBook OS are "macOS Catalina version 10.15.5". The old MacBook is "MacBook Pro 15" 2017". The new MacBook is "MacBook Pro 16", 2019".

Hope you can give me some advice to launch this app in the new MacBook. I cannot use a MacBook without this app! Thanks a lot!

Regards,
Robin Hu

**Comment by robingranite** [comment]
Hi @lwouis! 

Thanks for your quick response! It comes back! Wonderful! Thank you very much!

I referred the issue #127 and ticked the check box of AltTab in system Preferences -> Security & Privacy -> Screen Recording. Now It works!

I installed the new MacBook by using Migration Assistant. I guess something wrong happened of the permission of screen recording in this process.

Thank you!
Robin Hu
